test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -705,13 +705,18 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			initialProfile = profile.DeepCopy()
 		})
 
-		It("[disruptive] should set offline cpus after deploy PAO", func() {
+		FIt("[disruptive] should set offline cpus after deploy PAO", func() {
 			for _, node := range workerRTNodes {
 				onlineCPUCount, err := nodes.ExecCommandOnNode([]string{"nproc", "--all"}, &node)
 				Expect(err).ToNot(HaveOccurred())
 				onlineCPUInt, err := strconv.Atoi(onlineCPUCount)
 				Expect(err).ToNot(HaveOccurred())
 				Expect(onlineCPUInt).Should(BeNumerically(">=", 3))
+				onlineCPUsInt, err := strconv.Atoi(onlineCPUCount)
+				Expect(err).ToNot(HaveOccurred())
+				if onlineCPUsInt <= 8 {
+					Skip(fmt.Sprintf("This test needs more than 8 CPUs online to work correctly, current online CPUs are %s", onlineCPUCount))
+				}
```

Comment:
should we add skip out of the for loop, not sure if skip breaks the loop ?

Reply:
"Skip notifies Ginkgo that the current spec was skipped." I think so